policy/release/olm.rego

```diff
@@ -335,7 +378,7 @@ _csv_manifests contains manifest if {
 	manifest.kind == "ClusterServiceVersion"
 }
 
-# Verify allowed_registry_prefixes is a non-empty list of strings
+# Verify required_olm_features_annotations is a non-empty list of strings
 _rule_data_errors contains msg if {
```

Comment:
Newbie question... Is it more conventional to make a copy of the [entire block](https://github.com/enterprise-contract/ec-policies/blob/main/policy/release/olm.rego#L339-L354) to check `allowed_registry_prefixes` or, modify that block and pass it a list of _rule_data_keys consisting of `allowed_registry_prefixes` and `required_olm_features_annotations` ?

Reply:
I think in this case, a list of rule data keys makes more sense since both must conform to the same schema.

Also, we should make a change [here](https://github.com/release-engineering/rhtap-ec-policy/blob/main/data/rule_data.yml) to define the new rule data.

Reply:
I see that `required_olm_features_annotations` are missing from rhtap, but `allowed_registry_prefixes` [are there](https://github.com/release-engineering/rhtap-ec-policy/blob/74f112c8c00a03b079dc98f2ccb4ca127dcbd819/data/rule_data.yml#L5).

Also, looking at this more, since the time when the check for the features annotations were added [last December](https://github.com/enterprise-contract/ec-policies/pull/830) three of those were made optional:

```
features.operators.openshift.io/cnf
features.operators.openshift.io/cni
features.operators.openshift.io/csi
```

I can open a new separate ticket to update the rego checks for this, if one doesn't exist.

Reply:
I [opened a PR](https://github.com/release-engineering/rhtap-ec-policy/pull/34) for the missing features annotations rule data